### ISSUE TITLE
Warnings on 1.4.1

### ISSUE THREAD

**Issue by gideonsimpson** [SciML/DiffEqBiological.jl#192]
I just added and compiled this package, and it generated the following warnings:

```
WARNING: importing deprecated binding Compat.invokelatest into SymEngine.
WARNING: Compat.MathConstants is deprecated, use Base.MathConstants instead.
  likely near /Users/guardian/.julia/packages/SymEngine/RK5Ks/src/SymEngine.jl:6
WARNING: Compat.MathConstants is deprecated, use Base.MathConstants instead.
  likely near /Users/guardian/.julia/packages/SymEngine/RK5Ks/src/SymEngine.jl:6
WARNING: Compat.MathConstants is deprecated, use Base.MathConstants instead.
  likely near /Users/guardian/.julia/packages/SymEngine/RK5Ks/src/SymEngine.jl:6
WARNING: Compat.MathConstants is deprecated, use Base.MathConstants instead.
  likely near /Users/guardian/.julia/packages/SymEngine/RK5Ks/src/SymEngine.jl:6
WARNING: importing deprecated binding Compat.LinearAlgebra into SymEngine.
WARNING: importing deprecated binding Compat.Libdl into SymEngine.
WARNING: Compat.LinearAlgebra is deprecated, use LinearAlgebra instead.
  likely near /Users/guardian/.julia/packages/SymEngine/RK5Ks/src/dense-matrix.jl:162
WARNING: Compat.LinearAlgebra is deprecated, use LinearAlgebra instead.
  likely near /Users/guardian/.julia/packages/SymEngine/RK5Ks/src/dense-matrix.jl:166
WARNING: Compat.LinearAlgebra is deprecated, use LinearAlgebra instead.
  likely near /Users/guardian/.julia/packages/SymEngine/RK5Ks/src/dense-matrix.jl:168
WARNING: Compat.LinearAlgebra is deprecated, use LinearAlgebra instead.
  likely near /Users/guardian/.julia/packages/SymEngine/RK5Ks/src/dense-matrix.jl:168
┌ Warning: Use of `Poly` from v1.0 forward will require `using Polynomials.PolyCompat`
│   caller = Polynomials.PolyCompat.Poly(::Array{Double64,1}) at Poly.jl:26
└ @ Polynomials.PolyCompat ~/.julia/packages/Polynomials/ghqdF/src/polynomials/Poly.jl:26

```

**Comment by ChrisRackauckas** [comment]
Looks like this is an upstream issue with SymEngine.

**Comment by isuruf** [comment]
Fixed with SymEngine 0.8.1
